src/mongo/tree/MongoCollectionTreeItem.ts

```diff
@@ -25,6 +25,7 @@ export class MongoCollectionTreeItem implements IAzureParentTreeItem {
 	private _cursor: Cursor | undefined;
 	private _hasMoreChildren: boolean = true;
 	private _batchSize: number = DefaultBatchSize;
+	private readonly deferToShell = null;
```

Comment:
I'm finding this more confusing than simply commenting on the return values using `/**    @returns` etc.  Plus you don't seem to have used it.